var/spack/repos/builtin/packages/py-onnx-runtime/package.py

```diff
@@ -14,13 +14,16 @@ class PyOnnxRuntime(CMakePackage, PythonExtension):
     stays up to date with the ONNX standard with complete
     implementation of all ONNX operators, and supports all
     ONNX releases (1.2+) with both future and backwards
-    compatibility."""
+    compatibility.
+
+    Deprecated in favor of py-onnxruntime.
```

Comment:
Both the [GitHub](https://github.com/microsoft/onnxruntime) and [PyPI](https://pypi.org/project/onnxruntime/) are named onnxruntime, not onnx-runtime.